packages/dataviews/src/view-table.js

```diff
@@ -536,21 +536,17 @@ function ViewTable( {
 											minWidth: 20,
 										} }
 									>
-										<span className="dataviews-view-table__cell-content-wrapper">
```

Comment:
@oandregal was there a reason to remove this span? It's important for alignment in table layout, see: https://github.com/WordPress/gutenberg/pull/57804